src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import Movie from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  card: Movie;
+}
+
+export const MovieCard: React.FC<Props> = ({ card }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3" data-cy>
+        <img
+          data-cy="MovieImage"
+          src={card.imgUrl}
+          alt="movie"
+        />
+      </figure>
+    </div>
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="../images/imdb-logo.jpeg" alt="IMDB" />
```

Comment:
It doesn't work now 😢 
![image](https://user-images.githubusercontent.com/99403198/226101354-64b02f47-f710-4795-b610-cdc2ebcf8abe.png)